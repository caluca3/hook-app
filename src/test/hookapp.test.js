import { renderHook} from '@testing-library/react-hooks';
import HookApp from '../HookApp';
test('should increment counter', () => {
  const { result } = renderHook(() => HookApp())

  expect(result).toMatchSnapshot();
});
