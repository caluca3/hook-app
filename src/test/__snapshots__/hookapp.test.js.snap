// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should increment counter 1`] = `
Object {
  "all": Array [
    <React.Fragment>
      <div>
        HookApp
      </div>
    </React.Fragment>,
  ],
  "current": <React.Fragment>
    <div>
      HookApp
    </div>
  </React.Fragment>,
  "error": undefined,
}
`;
